### PR TITLE
textlint --fix README.md

### PR DESCRIPTION
**Pull request by sergio-alonso**

### CODE REVIEW

File: .textlintrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "rules": {
+	"eslint": {
+	    "configFile": "./.eslintrc"
```
Comment (feimosi) [inline .textlintrc]: Add also the following option:
`"langs": ["js", "ts", "jsx"]`

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+    "env": {
+	"es6": true,
+	"node": true,
+	"jasmine": true
```
Comment (feimosi) [inline .eslintrc]: `jasmine` and `node` can be removed.

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    "rules": {
+	"indent": ["error", 2, {"SwitchCase": 1}],
+	"no-else-return": "error",
+	"no-multi-spaces": "error",
+	"no-whitespace-before-property": "error",
+	"camelcase": "error",
+	"new-cap": "error",
+	"no-console": "error",
+	"comma-dangle": "error",
```
Comment (feimosi) [inline .eslintrc]: Set to `always`.

**Review by feimosi — CHANGES_REQUESTED** [review]
Awesome, thanks!

File: .eslintrc (hunk 3 above)
Comment (sergio-alonso) [reply .eslintrc]: always is not an option for all of the rules.

**Comment by sergio-alonso** [comment]
I suppose that it's a question of fine tune the rules to meet your needs.

**Comment by feimosi** [comment]
Sure, looks good :+1:

**Pull request merged**